README.md

```diff
@@ -2,13 +2,60 @@
 
 A NestJS module for managing environment variables easily and securely.
 
+<hr/>
+
+## What Difference From [official library](https://docs.nestjs.com/techniques/configuration)
+
+An `official library` ( [@nestjs/config](https://docs.nestjs.com/techniques/configuration) ) exists for managing environments in nestjs.
+
+However, we hoped that environment to be managed on a per `module basis`.
+
+We thought it would be nice to have some `simpler` and `easier` to `verify` and `type infer`.
+
+It was created by this background, and only `environment used per module` can be used.
+
+Instead of looking for and using varA, varB, varC... in configService, It can define own environment per module and use it.
```

Comment:
I have modified description as below. Please give your feedback :)

```
There is an official library ( @nestjs/config ) for managing  configurations for nestjs application.

However, we hoped that configurations to be managed by each module. We thought it would be nice to have simpler and easier way to validate configurations and infer type of them.

Here is where `@nestjs-library/config` kicks in. Instead of looking for configuration from global, you can define own configurations per module and use it as you wish.
```